### PR TITLE
Adding SP_DISPLAY_NAME and SP_DESCRIPTION when listing reciepts

### PR DESCRIPTION
**Pull request by IsuraD**

### CODE REVIEW

File: components/org.wso2.carbon.consent.mgt.endpoint/src/gen/java/org/wso2/carbon/consent/mgt/endpoint/dto/ConsentResponseDTO.java [hunk 1]
```diff
@@ -28,6 +28,12 @@
   
   
   private String state = null;
+  
+
```
Comment (ayshsandu) [inline components/org.wso2.carbon.consent.mgt.endpoint/src/gen/java/org/wso2/carbon/consent/mgt/endpoint/dto/ConsentResponseDTO.java]: remove unwanted new lines.

File: components/org.wso2.carbon.consent.mgt.endpoint/src/gen/java/org/wso2/carbon/consent/mgt/endpoint/dto/ConsentResponseDTO.java [hunk 2]
```diff
@@ -90,6 +96,30 @@ public void setState(String state) {
   }
 
   
+  /**
+   **/
+  @ApiModelProperty(value = "")
+  @JsonProperty("spDisplayName")
+  public String getSpDisplayName() {
+    return spDisplayName;
+  }
```
Comment (ayshsandu) [inline components/org.wso2.carbon.consent.mgt.endpoint/src/gen/java/org/wso2/carbon/consent/mgt/endpoint/dto/ConsentResponseDTO.java]: add new line after

File: components/org.wso2.carbon.consent.mgt.endpoint/src/gen/java/org/wso2/carbon/consent/mgt/endpoint/dto/ConsentResponseDTO.java (hunk 1 above)
Comment (IsuraD) [reply components/org.wso2.carbon.consent.mgt.endpoint/src/gen/java/org/wso2/carbon/consent/mgt/endpoint/dto/ConsentResponseDTO.java]: These are auto generated codes; swagger to cxf

File: components/org.wso2.carbon.consent.mgt.endpoint/src/gen/java/org/wso2/carbon/consent/mgt/endpoint/dto/ConsentResponseDTO.java (hunk 2 above)
Comment (IsuraD) [reply components/org.wso2.carbon.consent.mgt.endpoint/src/gen/java/org/wso2/carbon/consent/mgt/endpoint/dto/ConsentResponseDTO.java]: These are auto geneated codes; swagger to cxf

**Pull request merged**